packages/client/src/components/SignIn/index.js

```diff
@@ -0,0 +1,113 @@
+        remember: false,
+        showPassword: false,
+        errorText: ''
+    });
+
+    const handleChange = (prop) => (event) => {
+        if (prop === 'remember') {
+            setValues({ ...values, [prop]: !values.remember });
+        } else {
+            setValues({ ...values, [prop]: event.target.value });
+        }
+    }
+    const handleClickShowPassword = () => {
+        setValues({ ...values, showPassword: !values.showPassword });
+    };
+    const handleMouseDownPassword = (event) => {
+        event.preventDefault();
+    };
+
+    return (
+        <form>
+            { JSON.stringify(user) }
+            <br/>
+            <h1>Sign In</h1>
+
+            <div className="form-group">
+                <Text
+                    autoFocus = {true}
+                    id = 'email'
+                    placeholder = "Email"
+                    value = {values.email}
+                    onChange = {handleChange('email')}
+                    required = {true}
+                />
+            </div>
+
+            <div className="form-group">
+                <Text
+                    id = "password"
+                    placeholder = "Password"
+                    type = {values.showPassword ? 'text' : 'password'}
+                    value = {values.password}
+                    onChange = {handleChange('password')}
+                    required = {true}
+                    endAdornment = {{
+                        endAdornment:
+                            <InputAdornment position="end">
+                                <IconButton
+                                    onClick={handleClickShowPassword}
+                                    onMouseDown={handleMouseDownPassword}
+                                >
+                                    {values.showPassword ? <Visibility /> : <VisibilityOff />}
+                                </IconButton>
+                            </InputAdornment>
+                    }}
+                />
+            </div>
+
+            <div className="form-group">
+                <FormControlLabel
+                    control={
+                        <Checkbox
+                            id="checkbox"
+                            onChange={handleChange('remember')}
+                            value={values.remember}
+                            name="remember"
+                            color="primary"
+                        />
+                    }
+                    label="Remember Me"
+                />
+            </div>
+            <style type="text/css">
+                {`
+                    .btn-coved {
+                      background-color: #00568C;
+                      color: white;
+                    }
+                `}
+            </style>
+            <br />
+            <Button theme="default" size="md"
+                    onClick={() => {
+                        signin(values.email, values.password)
+                    }}
```

Comment:
You need to handle the promise here. Catch the failures and display a message to user. For example, if the username and password is wrong a message should be displayed notifying the user.

Reply:
Do I need to do anything if it succeeds (i.e. a `.then()`)?